libs/designsystem/loading-overlay/src/loading-overlay.component.scss

```
@@ -19,8 +19,7 @@
     display: flex;
     background: transparent;
     justify-content: center;
-    align-items: center;
-    justify-items: center;
+    place-items: center center;
```

Consider simplifying further (same value):
```suggestion
    place-items: center;
```